docker-build.sh

```diff
@@ -300,6 +307,11 @@ init_variables() {
 
   # split tags (to allow multiple comma-separated tags)
   IFS=, read -ra INPUT_IMAGE_TAG <<< "$INPUT_IMAGE_TAG"
+
```

Comment:
I'd rather move this condition as an `elif` insode `_set_namespace` (after the `elif` of `_is_gcloud_container_registry`.
Let me know if you want me to do it if it's not clear.

Thanks for the contribution!

Reply:
I appreciate it man!

I was not sure if it would be confusing to call `_set_namespace` just to keep `$NAMESPACE` empty for `_is_gcloud_artifact_registry`. However, I can move this condition 👍🏻 .

Also, this PR is not quite ready yet. The error I had seen via https://github.com/whoan/docker-build-with-cache-action/issues/158 makes me believe I may had done something incorrect since this error is thrown in [entrypoint.sh#L29-L32](https://github.com/whoan/docker-build-with-cache-action/blob/fac15f574a4d2286eff22a3a19ceab6847a4b00d/entrypoint.sh#L29-L32). I have not had a chance to test this PR yet, but I'll try to get to it at some point 🙂